src/VR.js

```diff
@@ -356,23 +354,10 @@ class VRDisplay extends EventEmitter {
   }
 }
 
-class SpatialEvent extends Event {
-  constructor(type, init = {}) {
-    super(type);
-
-    if (init.detail) {
-      for (const k in init.detail) {
-        this[k] = init.detail[k];
-      }
-    }
-  }
-}
-
-class FakeMesher extends EventTarget {
-  constructor(session) {
+class FakeMesher extends EventEmitter {
```

Comment:
We should move all of the VR fakers to a separate `FakeXR.js` so it doesn't get in the way of the standard binding code.

We could also probably not even `require` it in the main engine case, where we don't use fake XR devices at all.

Reply:
I agree, it seems out of place to include the fakes in the core engine (`index.js`, etc.).